### PR TITLE
add logkit roadmap

### PR DESCRIPTION
**Pull request by wonderflow**
## Changes
   
  - [ ] add logkit roadmap
 
## Reviewers

  - [ ] @lvheyang please review
  - [ ] @wangtuo  please review
  - [ ] @andrewei1316 

## Wiki Changes
 
  - options1...
  - options2...
    
## Checklist
   
   - [ ] Rebased/mergable
   - [ ] Tests pass
   - [ ] Wiki updated

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+1. 支持 kafka sender
+2. 支持 http sender
+3. logkit 统计信息持久化
+4. 更多需要的Transformer
+
+
+# 1.4.X
+
+1. 分布式logkit （raft）
```
Comment (lvheyang) [inline ROADMAP.md]: raft 指的是？master 的HA吗？

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# 1.3.X
+
+1. 支持 kafka sender
+2. 支持 http sender
```
Comment (lvheyang) [inline ROADMAP.md]: http reader 需要么？指定端口接受数据

**Comment by andrewei1316** [comment]
补充： metric 功能模块的完善

File: ROADMAP.md (hunk 2 above)
Comment (wonderflow) [reply ROADMAP.md]: 需要

File: ROADMAP.md (hunk 1 above)
Comment (wonderflow) [reply ROADMAP.md]: 全搞成raft node

Comment (wonderflow) [reply ROADMAP.md]: 也是 master的HA

**Pull request merged**